react-app/src/index.css

```diff
@@ -1 +1,4 @@
 /* TODO Add site wide styles */
+* {
```

Comment:
Rather than selecting any element, set this to `body` so elements just inherit from it. This just ends up looking a bit cleaner and makes overriding the style easier.